locust/core.py

```diff
@@ -73,34 +73,38 @@ class Locust(object):
     example when load testing an HTTP system, you probably want to use the 
     :py:class:`HttpLocust <locust.core.HttpLocust>` class.
     """
-    
+
     host = None
     """Base hostname to swarm. i.e: http://127.0.0.1:1234"""
-    
+
     min_wait = 1000
     """Minimum waiting time between the execution of locust tasks"""
-    
+
     max_wait = 1000
     """Maximum waiting time between the execution of locust tasks"""
-    
+
+    random_execute = True
+    """Random execute flag, default is True"""
+
     task_set = None
     """TaskSet class that defines the execution behaviour of this locust"""
-    
+
     stop_timeout = None
     """Number of seconds after which the Locust will die. If None it won't timeout."""
 
     weight = 10
     """Probability of locust being chosen. The higher the weight, the greater is the chance of it being chosen."""
-        
+
     client = NoClientWarningRaiser()
     _catch_exceptions = True
-    
+
     def __init__(self):
         super(Locust, self).__init__()
-    
+        self.task_set_instance = self.task_set(self)
```

Comment:
`self.task_set_instance = self.task_set(self)` is failing in Python 2.7 with `TypeError: 'NoneType' object is not callable`